influx-reports/influx-report.sh

```diff
@@ -0,0 +1,6 @@
+PYTHON_PATH="${WORKSPACE}/.venv/bin/python"
+REPORTS_FOLDER="${WORKSPACE}/test_results"
+INFLUX_DOWNTIME_FOLDER="$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
The output data gives figures for uptime, the use of the term downtime in this shell script might be misleading.